buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/IntegrationTest.kt

```diff
@@ -30,6 +31,10 @@ import org.gradle.gradlebuild.BuildEnvironment
  */
 @CacheableTask
 open class IntegrationTest : DistributionTest() {
+
+    @get:Nested
+    val distributionSamples: GradleDistributionSamples = GradleDistributionSamples(project, gradleInstallationForTest.gradleUserHomeDir)
```

Comment:
I guess you confused `gradleUserHome` and `gradleHome` here. `gradleUserHome` is `~/.gradle` and `gradleHome` is the root project of gradle distribution, like `JAVA_HOME`.

Reply:
Yeah, I fixed it.